src/api/request.js

```diff
@@ -1,13 +1,10 @@
 import axios from 'axios';
-import store from '../store';
 
 export default {
   $http(header = {}) {
     return axios.create({
       baseURL: process.env.VUE_APP_ROOT_API,
       headers: {
-        ...(store.getters.authToken()
-          ? { Authorization: `Bearer ${store.getters.authToken()}` } : {}),
```

Comment:
Perhaps its would be better maintain authorization token if user is logged, is there some problem if use this approach?

Reply:
It was removed because it is set as a common global config of axios, so it is not needed to pass through every API calling.